### PR TITLE
Notify user when no APIs found with selected status

### LINKED ISSUE #2396: Filter by 'Life Cycle phase': Notify user when no APIs found with selected status
# Steps 
1.Visit https://staging.apinf.io v 0.42.0 
2.Login  
3.Go to API catalog 
4.Filter by 'Life Cycle' status (eg. deprecated)  

# Findings  
- When no 'APIs' found by selected status , the sorting result is shown as: 'Filtered by "Deprecated" "Clear All"
- Sort result could notify user eg. "No APIs found with the selected life cycle status"      

# Screenshot  
sort_results_lifecycle_status

### PR DESCRIPTION
**Pull request by Vellames**
Closes #2396 

Added a notify when no APIs found with selected status.

### CODE REVIEW

**Comment by Vellames** [comment]
@saralavanip, @bajiat ready for review

**Comment by Vellames** [comment]
One more thing, i used the google translator to generate the Suomi string, i am not sure if the text are correct

**Comment by marla-singer** [comment]
@Vellames Don't provide Finish text. It will be added by @bajiat in a translation round. A developer role is to create an i18n tag for the English language.

**Comment by Vellames** [comment]
@marla-singer Ok, thank you for the information.

**Comment by Vellames** [comment]
@marla-singer translation removed

File: apinf_packages/core/lib/i18n/en.i18n.json [hunk 1]
```diff
@@ -71,7 +71,7 @@
   "apiCardPin_unfeaturedSuccessfully": "API card removed from featured list successfully",
   "apiCatalog_apisCount": "(__count__ api)",
   "apiCatalog_apisCount_plural": "(__count__ apis)",
-  "apiCatalog_noApis_Found": "No APIs found with the selected life cycle status",
+  "apiCatalog_noApis_Found": "",
```
Comment (marla-singer) [inline apinf_packages/core/lib/i18n/en.i18n.json]: @Vellames Sorry for disinformation. I meant a developer create i18n tag **and write the English text for this tag**. So back the English text again :grimacing:

Comment (Vellames) [reply apinf_packages/core/lib/i18n/en.i18n.json]: @marla-singer no problem :)
Inserted again

**Pull request merged**